test/fixtures/userStatus/userStatus.js

```diff
@@ -0,0 +1,56 @@
+const userStsDataForOooState = (userId) => {
+  return {
+    userId,
+    currentStatus: {
+      until: "1669256009000",
```

Comment:
@heyrandhir If these are strings, would we be able to perform range search queries on the dates?

Should these be numeric instead? And be indexable?

Reply:
I need to check whether we can perform a range search on the date field. Yeah, it would be better to store it as numeric. I have updated the code to store it as numeric.